pkg/storage/filesystem/device.go

```diff
@@ -68,10 +70,38 @@ func (i *DeviceInfo) Used() bool {
 	return len(i.Label) != 0 || len(i.Filesystem) != 0
 }
 
-func (d *DeviceInfo) Type() (zos.DeviceType, error) {
+// DetectType returns the device type according to seektime
+func (d *DeviceInfo) DetectType() (zos.DeviceType, error) {
 	return d.mgr.Seektime(context.Background(), d.Path)
 }
 
+// SetType sets the device type to the disk
+func (d *DeviceInfo) SetType(typ pkg.DeviceType) error {
+	if err := os.WriteFile(filepath.Join("/mnt", d.Name(), ".seektime"), []byte(typ), 0644); err != nil {
```

Comment:
let's extract the first argument of WriteFile into a varilable to improve the readability

Reply:
actually I believe we shouldn't use `.seektime` extension, as it denotes the results of seektime (rpm/seektime) instead of actual harddisk type string, however if it's considered as "the output of seektime" tool, I'm good with that